get-authors.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env sh
+
+REPO_URL=$1
+EMAIL_DOMAIN=$2
+OUTPUT_PATH=${3:-authors.txt}
+
```

Comment:
Nice!  I've updated the script to use your much neater version, with a few tweaks to make it a little easier (for me) to read.  I've also added usage information.